cogs/logs.py

```diff
@@ -28,6 +28,48 @@ def __init__(self, bot):
         # We use this a lot, might as well get it once
         escaped_name = self.bot.escape_message(member)
 
+        # Attempt to correlate the user joining with an invite
+        with open("data/invites.json", "r") as f:
+            invites = json.load(f)
+
+        real_invites = await member.guild.invites()
+
+        # Add unknown active invites. Can happen if invite was manually created
+        for invite in real_invites:
+            if invite.id not in invites:
+                invites[invite.id] = { "uses": 0, "url": invite.url, "max_uses": invite.max_uses, "code": invite.code }
```

Comment:
1) This line is over 80 chars, can you split it?
2) Can you remove the spaces after `{` and before `}`?